src/wms/src/test/java/org/geoserver/wms/WMSCascadeTestSupport.java

```diff
@@ -185,6 +237,57 @@ private void setupWMS110Layer() throws MalformedURLException, IOException {
                 new URL(mockPNGUrl), new MockHttpResponse(pngRoadsImage, "image/png"));
         wms11Client.expectGet(
                 new URL(mockJpegUrl), new MockHttpResponse(gifRoadsImage, "image/gif"));
+
+        // SET UP layer group of two cascaded wms layers
+        WMSLayerInfo group_lyr_1 = cb.buildWMSLayer("group_lyr_1");
+        group_lyr_1.setName("group_lyr_1");
+        group_lyr_1.reset();
+        group_lyr_1.setRespectMetadataBBox(true);
+        // group_lyr_1.setForcedRemoteStyle("line_group");
+        getCatalog().add(group_lyr_1);
+        LayerInfo layer1 = cb.buildLayer(group_lyr_1);
+        getCatalog().add(layer1);
+
+        WMSLayerInfo group_lyr_2 = cb.buildWMSLayer("group_lyr_2");
+        group_lyr_2.setName("group_lyr_2");
+        group_lyr_2.reset();
+        group_lyr_2.setRespectMetadataBBox(true);
+        // group_lyr_1.setForcedRemoteStyle("line_group2");
+        getCatalog().add(group_lyr_2);
+        LayerInfo layer2 = cb.buildLayer(group_lyr_2);
+        getCatalog().add(layer2);
+
+        LayerGroupInfo group = getCatalog().getFactory().createLayerGroup();
+        // LayerGroupInfoImpl group = new LayerGroupInfoImpl();
+        // (LayerGroupInfoImpl) getCatalog().getFactory().createLayerGroup();
+
+        // group.setId("casc_group");
+        group.setName("cascaded_group");
+        group.getLayers().add(layer1);
+        group.getLayers().add(layer2);
+        try {
+            cb.calculateLayerGroupBounds(group);
+        } catch (Exception e) {
```

Comment:
Just let it throw the exception then?

Reply:
fail() is called in next line

Reply:
It's un-necessary and hides the actual stack trace, so no, it's not.